docs/OnTrack/Jupyter Notebook/Docker-Documentation-Research-T1-2022.md

```diff
@@ -1,32 +1,67 @@
 # Docker Documentation and Research (WIP) T1-2022
 
-- [Back to Jupyter Notebook Documentation Index](Index.md)
+[Back to Jupyter Notebook Documentation Index](Index.md)
 
-## Conversion Relating to Docker Processes
+## Intro
+
+As we know, the Jupyter Notebook conversion feature will happen via processes that occur inside
+Docker containers. In this document we will discuss the main architecture of this structure in
+relation to OnTrack.
+
+OnTrack is deployed with two main containers that will always be running:
+
+- the frontend container which hosts the logic behind the user interface of OnTrack (mostly
+  unrelated to this feature)
+- the backend container which hosts the logic related to database interactions and other processes
+
+To achieve the Jupyter conversion feature, we will be using a new container that contains all of the
```

Comment:
Maybe "the Jupyter **Notebook** conversion feature...", just so the reader understands that is what you are talking about. Whilst it may not be the case of every reader, some readers might see 'Jupyter Notebook' and 'Jupyter' and think they are two separate things. Just a suggestion though.